package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@paprika/paprika",
   "description": "",
   "scripts": {
+    "readme": "node scripts/readmeFiles.js",
```

Comment:
Can we run this script automatically during the build?

Reply:
yes, I will do that next PR. has to be on `prepare`.